products/views.py

```diff
@@ -1,3 +1,67 @@
-from django.shortcuts import render
+from django.views       import View
+from django.http        import JsonResponse
+from django.db.models   import Q
+
+from products.models    import (
+    OptionProduct,
+    Product,
+    Material,
+    PhoneType,
+    AirpotType,
+    ShoeSize,
+    ProductCategory,
+    DetailImage
+)
+
+class ProductDetailView(View):
+       def get(self, request, product_id):
+        try:
+            product          = Product.objects.get(id = product_id)
+            option_products  = OptionProduct.objects.filter(product_id = product_id)
+
+            if OptionProduct.objects.filter(Q(product_id=product_id) & Q(shoe_size_id=None) & Q(phone_type_id=None) & Q(airpot_type_id=None)):
+                option_existence = False
+
+            else: 
+                option_existence = True
```

Comment:
`exists()`를 사용하여 아래와 같이 작성할 수 있을 것 같습니다.
```suggestion
            option_existence = True if OptionProduct.objects.filter(~Q(shoe_size = None) | ~Q(phone_type = None) | ~Q(airpot_type = None)).exists() else False

```